DATA/SOLAR/ASTEROIDS/bw02_minefield.ini

```diff
@@ -3,7 +3,7 @@ file = solar\asteroids\ice_shapes.ini
 
 [Field]
 cube_size = 400
-fill_dist = 2600
```

Comment:
Based on which exact fields did you determine that the fill distance should be 2800?

Reply:
I checked again. Might've been mistaken by files I had in my own mod.